lib/lte_link_control/lte_lc.c

```diff
@@ -1525,14 +1399,16 @@ int lte_lc_neighbor_cell_measurement(enum lte_lc_neighbor_search_type type)
 
 int lte_lc_neighbor_cell_measurement_cancel(void)
```

Comment:
This is merged but above function has at_cmd_write (due to PR that was merged while this was ongoing):

```
int lte_lc_neighbor_cell_measurement(enum lte_lc_neighbor_search_type type)
...
	if (type == LTE_LC_NEIGHBOR_SEARCH_TYPE_EXTENDED_LIGHT) {
		err = at_cmd_write("AT%NCELLMEAS=1",  NULL, 0, NULL);
	} else if (type == LTE_LC_NEIGHBOR_SEARCH_TYPE_EXTENDED_COMPLETE) {
		err = at_cmd_write("AT%NCELLMEAS=2",  NULL, 0, NULL);
	} else {
		/* Defaulting to use LTE_LC_NEIGHBOR_SEARCH_TYPE_DEFAULT */
		err = at_cmd_write("AT%NCELLMEAS",  NULL, 0, NULL);
	}

```

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/5999